resources/definitions/zonestar_z5x.def.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "ZONESTAR Z5X",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "ZONESTAR",
+        "manufacturer": "ZONESTAR",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0],        
+        "machine_extruder_trains":
+        {
+            "0": "zonestar_Z5X_extruder"

```

Comment:
According to [this sales page](https://www.madethebest.com/products/zonestar-z5x-series-large-high-precision-resolution-ultra-silent-easy-install-optional-dual-extrusion-auto-mix-color-3d-printer) the printer has two extruders. Are you missing one?

Reply:
Z5X has only one extruder, please keep these settings, thanks!

Reply:
I beg to differ. It says it right there on the page with great pride:

> 2-IN-2-OUT(Z5XR2): The printer has 2 extrusion feeders and 2 nozzle. It can use one nozzle to print the main object and another nozzle to print support filament, for exmple, the main extruder to print PLA and another nozzle to print PVA. Of couse, you can use it to print two color PLA or (ABS) filament, etc.
> 
> 2-IN-1-OUT(Z5XM2): The printer has 2 extrusion feeders and 1 nozzle. The filament will be mixed in the hotend and then flow out from one nozzle, you can use it to mix two colors filament together and become a new color, usually we suggest you use only one type filament in the same time. With ZONESTAR's patented technology, you can easily set the mixed ratio of two filaments on the control panel when printing from SD card, and even let the machine automatically adjust the mixed ratio to become a multi-color object.

If this printer does not have 2 extruders, what "patented technology" is that website referring to that tunes the mixing ratio?